### PR TITLE
Enabled bg playback on Android to 50% percent of users on nightly channel(production)

### PR DESCRIPTION
**Pull request by SergeyZhukovsky**
Resolves https://github.com/brave/brave-variations/issues/384

### CODE REVIEW

**Review by tmancey — APPROVED** [review]
LGTM++

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/385#issuecomment-1275283460.

**Pull request merged**
